src/content/developers/docs/gas/index.md

```diff
@@ -74,7 +74,43 @@ For example if you put a gas limit of 50,000 for a simple ETH transfer, the EVM
 
 ## What is Gas price? {#what-is-gas-price}
 
-Gas price refers to the amount of Ether you are willing to pay for every unit of gas, and this is usually measured in 'gwei'.
+Gas price refers to the amount of Ether you are willing to pay for every unit of gas, and this is usually measured in 'gwei'. Prior to
+[the London update](history/#london), you specify in the transaction how much you are willing to pay per gas, and you pay exactly that amount. 
+Different transactions in the same block can have very different gas prices.
+
+### The London Update {#gas-price-london-update}
+
+Starting with the London update, every block has a base fee, the minimum per gas price for inclusion in this block. The base fee is calculated by a formula
+that compares the size of the previous block (the amount of gas used for all the transactions) with a target size. If the block size is higher than the target, there
+is more demand for inclusion in the blockchain than supply so the base fee, is increased. If the block size is lower than the target then there is more supply 
+of block space than demand, so the base fee is decreased. This base fee is "burned", removed from circulation.
+
+Transactions can either specify a gas price using the old mechanism, or specify two other parameters:
+
+-  Maximum Fee per Gas: The maximum gas price the transaction can be charged.
+-  Maximum Priority Fee per Gas (a.k.a. Tip): The maximum priority fee the transaction signer is willing to pay the miner per gas to be included.
+   If the base fee plus this amount is less than the maximum fee per gas, this is the priority fee. Otherwise, the priority fee is the maximum
+   fee minus the base fee.
+
+For example, imagine a block with a base fee of 100 gwei. The pool of available transactions contains these transactions:
+
+| ID     | Maximum Fee per Gas  | Maximum Priority Fee per Gas | Actual Priority Fee | Actual Gas Price |Remarks |
+| ------ | ------------------:  | ---------------------------: | ------------------: | ---------------: |------- |
+| A      | 90 gwei              | 90 gwei                      | N/A                 | N/A              | This transaction is not going in the block |
+| B      | 200 gwei             |  5 gwei                      | 5 gwei              | 105 gwei         | The priority fee is the maximum priority fee |
+| C      | 120 gwei             | 30 gwei                      | 20 gwei             | 120 gwei         | The priority fee is the maximum (total) fee minus the base fee |
+| D      | gas price = 200 gwei | gas price = 200 gwei         | 100 gwei            | 200 gwei         | Transactions that specify gas price are charged the full amount |
```

Comment:
Did you mean to include "gas price = " on only two of these cells?

Reply:
Yes, because only transaction D is old style and actually has a gas price field. I added to the paragraph that introduces the table to clarify.